stockflux-securities-master/src/services/SecuritiesService.js

```diff
@@ -41,19 +41,29 @@ export async function postSecurity(security) {
     `${options.customData.apiBaseUrl}/admin/securities`,
     fetchOptions
   );
-  const json = await response.json();
-
-  if (response.ok) {
-    return json;
+  if (response.status === 201) {
```

Comment:
I think it might be better to use [`response.ok`](https://developer.mozilla.org/en-US/docs/Web/API/Response/ok) here instead of explicitly checking the status code. This will be `true` if the status code falls within 200 - 299 and `false` otherwise. In this scenario nothing would change, but if we changed the success response code in the API from `201` to `200` or if it was originally `200` and we wanted a more descriptive `201` then your service would work regardless, whereas here it would fail